crates/ruff_linter/src/settings/types.rs

```diff
@@ -84,14 +81,6 @@ impl PythonVersion {
         }
     }
 
-    pub const fn major(&self) -> u8 {
-        self.as_tuple().0
-    }
-
-    pub const fn minor(&self) -> u8 {
-        self.as_tuple().1
-    }
-
     /// Infer the minimum supported [`PythonVersion`] from a `requires-python` specifier.
     pub fn get_minimum_supported_version(requires_version: &VersionSpecifiers) -> Option<Self> {
```

Comment:
this method only seems to be used from the `ruff_workspace` crate so it might make sense to move it there